chroma-manager/tests/integration/installation_and_upgrade/test_update_with_yum.py

```diff
@@ -59,24 +39,30 @@ def test_yum_update(self):
                                                          of_type='UpdatesAvailableAlert'))
 
         # Fully update all installed packages on the server
-        for server in self.TEST_SERVERS:
+        for server in self.config_servers:
             self.remote_operations.yum_update(server)
             kernel = self.remote_operations.default_boot_kernel_path(server)
             self.assertGreaterEqual(kernel.find("_lustre"), 7)
 
-        # Start the filesystem back up
-        filesystem = self.get_filesystem_by_name(self.fs_name)
-        self.start_filesystem(filesystem['id'])
-
+    @skip("Repos can't really be retired until at least an n+1 release")
     def test_no_retired_repos(self):
         "Test that no retired repos exist after an upgrade"
-        retired_repos = ['xeon-phi-client']
+        # TODO: this really needs to be more dynamic than using
+        #       repos that would have been retired many re-
+        #       leases ago
+        retired_repos = ['robinhood']
         current_repos = self.remote_operations.get_chroma_repos()
         for repo in retired_repos:
             self.assertFalse(repo in current_repos, "Unexpectedly found repo '%s' in %s" % (repo, current_repos))
 
+    # something we can run to clear the storage targets since this
+    # test class doesn't use setUp()
+    def test_clean_linux_devices(self):
```

Comment:
not sure why this should be a test, couldn't we just call self.cleanup_linux_devices where we need to?

Reply:
I'm assuming there's a reason why we don't need to clear zfs devices?

Reply:
Because the "test" is called from the manual upgrade procedure at https://github.com/intel-hpdd/intel-manager-for-lustre/pull/370/files#diff-33aef58adb4d563e360da3ef9403c698R124.  This test is the call into the the python framework from outside of it.

The test is only using ldiskfs.

This not meant to be the perfection of the upgrade test.  It's mean to get it running again and testing the manual upgrade procedure as it is documented so that we don't regress.

The whole process of package installation and upgrade needs to be redesigned and along with it the upgrade test.